### PR TITLE
[16.0][MIG] pos_product_multi_barcode: Migration to 16.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**

### CODE REVIEW

**Comment by luke-stdev001** [comment]
@nguyenminhchien ,

Happy to do a functional review on this as a bystander, do you have usage steps for this in the README.md?

**Comment by nguyenminhchien** [comment]
@luke-stdev001 
Hello, the module [product_multi_barcode](https://github.com/OCA/stock-logistics-barcode/tree/16.0/product_multi_barcode) allows you to add many barcodes for a product (have a look at the attached image below).
the POS feature allows you to select a product (from the pos screen) via scanning its barcode.
now, this module allows you to scan any of barcodes you define in product form view
image

File: pos_product_multi_barcode/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Ooops404 <https://ooops404.com>
```
Comment (simahawk) [inline pos_product_multi_barcode/readme/CONTRIBUTORS.rst]: funny that the original author @PierrickBrun is not here... can you fix this pls?

**Review by simahawk — APPROVED** [review]
LG, just code review

**Review by cyrilmanuel — APPROVED** [review]

**Pull request merged**